src/targets/gpu/kernels/include/migraphx/kernels/algorithm.hpp

```diff
@@ -44,6 +44,12 @@ struct greater
     }
 };
 
+template <class T>
+struct multiplies
+{
+    constexpr T operator()(const T& lhs, const T& rhs) { return lhs * rhs; }
+};
```

Comment:
There is already a `product` class in ops.hpp that can be used instead.

Reply:
Done.